Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -172,7 +175,7 @@
     if (auto* textAttachment = dynamic_objc_cast<NSTextAttachment>(value))
         return { { { RetainPtr { textAttachment } } } };
     if (auto* font = dynamic_objc_cast<NSFont>(value))
-        return { { { RetainPtr { font } } } };
+        return { { { Font::create(FontPlatformData(toCTFont(font), font.pointSize)) } } };
```

Comment:
Ditto

Reply:
We would have to first cast from id to a UIFont, then call bridge_cast, which would require a bunch of platform-specific stuff to be added.  I'm going to use __bridge instead for this PR and if further cleanup is desired it can be done separately.